Resources/Prototypes/Entities/Objects/Tools/flare.yml

```diff
@@ -78,3 +78,6 @@
         startValue: 8.0
         endValue: 1.0
         property: Radius
+  - type: Tag
+    tags:
+    - Trash
```

Comment:
The flare already has a 
```yaml
  - type: Tag
    tags:
    - Flare
```
further up

Reply:
> 

Yeah turns out I rushed this instead of checking tags then had to run. Will see if it passes now.